Remove unnecessary ATen/core/EnableNamedTensor.h

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31117 Remove unnecessary ATen/core/EnableNamedTensor.h**
* #31116 Remove all remaining usages of BUILD_NAMEDTENSOR

After this diff, we will have completely removed the named tensor
feature flagging. This means that named tensors are always on and that
there is no mechanism to turn them off. There should be no more follow-up
diffs.

I performed the deletion of the header with
```
find . -type f -print0 | xargs -0 sed -i '/#include
<ATen\/core\/EnableNamedTensor.h>/d'
```

Test Plan:
- wait for CI

Differential Revision: [D18934952](https://our.internmc.facebook.com/intern/diff/D18934952)

zou3519 committed:
Remove unnecessary ATen/core/EnableNamedTensor.h

After this diff, we will have completely removed the named tensor
feature flagging. This means that named tensors are always on and that
there is no mechanism to turn them off. There should be no more follow-up
diffs.

I performed the deletion of the header with
```
find . -type f -print0 | xargs -0 sed -i '/#include
<ATen\/core\/EnableNamedTensor.h>/d'
```

Test Plan:
- wait for CI

[ghstack-poisoned]

zou3519 committed:
Update on "Remove unnecessary ATen/core/EnableNamedTensor.h"


After this diff, we will have completely removed the named tensor
feature flagging. This means that named tensors are always on and that
there is no mechanism to turn them off. There should be no more follow-up
diffs.

I performed the deletion of the header with
```
find . -type f -print0 | xargs -0 sed -i '/#include
<ATen\/core\/EnableNamedTensor.h>/d'
```

Test Plan:
- wait for CI

[ghstack-poisoned]

nairbv commented on aten/src/ATen/core/EnableNamedTensor.h:
```diff
@@ -1,2 +0,0 @@
-#pragma once
-// We are working on removing the BUILD_NAMEDTENSOR flag from the codebase.
```
it's a bit unclear to me from github, is this removing the file or just the contents of the file?

zou3519 replied:
This is removing the file

zou3519 replied:
![image](https://user-images.githubusercontent.com/5652049/70651392-b3d68b00-1c1e-11ea-90d2-2b25895a1c72.png)

zou3519 committed:
Update on "Remove unnecessary ATen/core/EnableNamedTensor.h"


After this diff, we will have completely removed the named tensor
feature flagging. This means that named tensors are always on and that
there is no mechanism to turn them off. There should be no more follow-up
diffs.

I performed the deletion of the header with
```
find . -type f -print0 | xargs -0 sed -i '/#include
<ATen\/core\/EnableNamedTensor.h>/d'
```

Test Plan:
- wait for CI

Differential Revision: [D18934952](https://our.internmc.facebook.com/intern/diff/D18934952)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit b31cc46:
* **1/1** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=b31cc46353784f70953dbf9de8cf41b0b2a24a0f).

### 1 failure *not* recognized by patterns:
| Job | Step |
| --- | --- |
| ![CircleCI](https://avatars0.githubusercontent.com/ml/7?s=12) <sup>pytorch_windows_build</sup> | <sup>Build</sup> |


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

zou3519 commented:
I reran the windows build and it succeeded: https://app.circleci.com/jobs/github/pytorch/pytorch/3923679 so the build looks flaky.
